javascript/prototypingCharacterClasses.md

```diff
@@ -0,0 +1,56 @@
+#Prototype Your (Character) Class#
+
+Continuing in using Javascript to build D&D characters, I decided to use prototypes.
```

Comment:
[This](https://github.com/thoughtbot/til/blob/master/javascript/funWithFunctions.md); not really a series; beyond that it's related n that I wrote it, is D&D themed and Javascript based.